sig-contributor-experience/triage-team/triage.md

```diff
@@ -0,0 +1,43 @@
+# Issue and PR Management
+
+## Purpose
+
+Speed up management of issues and PRs labeled with `sig/contributor-experience`.
+
+The SIG Contributor Experience [issues](https://github.com/kubernetes/community/labels/sig%2Fcontributor-experience) and [PRs](https://github.com/kubernetes/community/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+label%3Asig%2Fcontributor-experience+) are labeled with `sig/contributor-experience`.
+
+Following are few example searches on issue and PR:
+* [Open issues](https://github.com/kubernetes/community/labels/sig%2Fcontributor-experience)
+* [Open issues for a specific milestone](https://github.com/kubernetes/community/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3Asig%2Fcontributor-experience+milestone%3AMay)
+* [Open PRs](https://github.com/kubernetes/community/pulls?utf8=%E2%9C%93&q=is%3A+pr+is%3Aopen+label%3Asig%2Fcontributor-experience+) 
+* [Open PRs for a specific milestone](https://github.com/kubernetes/community/pulls?utf8=%E2%9C%93&q=is%3Apr+label%3Asig%2Fcontributor-experience+milestone%3AMay+is%3Aopen)
+
+## Scope
+
+Manage incoming issues and PRs that are already identified and labeled with the `sig/contributor-experience` particularly in the [Kubernetes community repository](https://github.com/kubernetes/community) but in general across various Kubernetes related GitHub repositories like [Kubernetes repository](https://github.com/kubernetes/kubernetes/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Asig%2Fcontributor-experience).
+
+## People involved
+
+Everyone is welcome to help manage issues and PRs but the work and responsibilities discussed in this document is created with [Triage Team](TODO-describe the team in a separate doc and add link) and [Triage Captain](TODO-describe the team in a separate doc and add link) role in mind.
+
+## Work involved
+
+The main work is to keep track of issues and PRs, ping people, add appropriate labels as needed and report the overall status to the SIG Leads.
+
+- **Validate if an issue is a bug**: Validate if the issue is indeed a bug by following these [instructions](https://github.com/kubernetes/community/blob/master/contributors/guide/issue-triage.md#validate-if-the-issue-is-bug).
+
+- **Determine it belongs to contributor-experience SIG**: An issue can be incorrectly labeled with the `sig/contributor-experience` label. If you think an issue does not belong to the `sig/contributor-experience`, add a comment stating so with a reason.
```

Comment:
Provide an example? eg:
```
/remove-sig contributor-experience
I don't think this is a SIG contributor experience issue because .... 
If you think this is incorrect, please add back with /sig contributor-experience
```
or something to that effect (links to actual examples also fine)

Reply:
Sure, that sounds great!